### PR TITLE
Add alert for the datasource (local JSON or Firebase) for main and camper controllers

### PR DESCRIPTION
**Pull request by artitudinale1**
Insert a debug alert tell on every template if working locally or with firebase 
rebase to work locally on first checkout - that's why maybe @silverwedding  couldn't see changes...- I commented out wrong stuff in camper.controller.js ???

this is for #17

### CODE REVIEW

**Comment by artitudinale1** [comment]
@chris-alexander I know you working on switcher but can we merge this in mean time, it will be easier for every one rather than delete commenting eveytime.

File: dev/app/main/main.controller.js [hunk 1]
```diff
@@ -1,29 +1,19 @@
...
   'use strict';
   app.controller('mainC', mainC)
 
   function mainC($scope, getData) {
-    // CONTROLLER BODY FOR AJAX CALL TO LOCAL JSON - UNCOMMENT THIS IF WANT WORK WITH LOCAL JSON
-    // THE NAME OF THE FIRST CAMPERS SHOULD BE "WORKING FROM LOCAL JSON"
-    // IN shared/service.js YOU HAVE TO UNCOMMENT THE RIGHT SERVICE and campers.controller.js YOU HAVE TO UNCOMMENT THE RIGHT BODY CONTROLLER
-
-    
+    if (ENV_isFirebaseOff){
```
Comment (chris-alexander) [inline dev/app/main/main.controller.js:6]: Where is this variable defined?

Comment (artitudinale1) [inline dev/app/main/main.controller.js:6]: is in shared/service.js :)

Comment (chris-alexander) [inline dev/app/main/main.controller.js:6]: oh yeah now I see it! :smile: does changing the variable work?

Comment (artitudinale1) [inline dev/app/main/main.controller.js:6]: it does ...otherwise I wouldn't have create PR :)

**Comment by chris-alexander** [comment]
Closes #17

**Pull request merged**